Source/Tools/FEXLoader/FEXLoader.cpp

```diff
@@ -194,15 +194,49 @@ void RootFSRedirect(fextl::string* Filename, const fextl::string& RootFS) {
   }
 }
 
+FEX::Config::PortableInformation GetIsPortable() {
```

Comment:
This sounds cheaper than it is. Something like `ReadPortabilityInformation` reflects better that this reads the environment and filesystem.

Reply:
Updated name of the function.